### PR TITLE
Contract Explorer: Contract info

### PR DESCRIPTION
**Pull request by quietbits**
- Smart Contracts page and main navigation link
- Contract Info section
- Top header: add another breakpoint for resizing
- Created `SwitchNetworkButtons` component to be used where we have buttons to switch the network
- Also created `MessageField` component
- Updated Fee Bump test to make it less flaky (it was failing for me locally once in a while)

You can use `CBP7NO6F7FRDHSOFQBT2L2UWYIZ2PU76JKVRYAQTG3KZSQLYAOKIF2WB` contract ID to test the response on Mainnet.

<details>
<summary>Screenshots</summary>
<hr />

image

image

image
</details>

### CODE REVIEW

File: src/app/(sidebar)/account/saved/page.tsx [hunk 1]
```diff
@@ -131,25 +132,10 @@ export default function SavedKeypairs() {
...
-          <Button
-            variant="tertiary"
-            size="sm"
-            onClick={() => {
-              getAndSetNetwork("testnet");
-            }}
-          >
-            Switch to Testnet
-          </Button>
+          <SwitchNetworkButtons
```
Comment (jeesunikim) [inline src/app/(sidebar)/account/saved/page.tsx:135]: i love this

File: src/types/types.ts [hunk 2]
```diff
@@ -351,3 +351,35 @@ export interface SavedRpcMethod extends LocalStorageSavedItem {
...
+    package?: string;
+    make?: string;
+    ts?: number;
+  };
+  versions?: number;
+  salt?: string;
+  asset?: string;
+  code?: string;
+  issuer?: string;
+  functions?: {
```
Comment (jeesunikim) [inline src/types/types.ts:380]: i don't see `functions` from contracts-api.yml's ContractInfo schema

File: src/types/types.ts [hunk 3]
```diff
@@ -351,3 +351,35 @@ export interface SavedRpcMethod extends LocalStorageSavedItem {
...
+  contract: string;
+  created: number;
+  creator: string;
+  account?: string;
+  payments?: number;
+  trades?: number;
+  wasm?: string;
+  storage_entries?: number;
+  validation?: {
+    status?: "verified" | "unverified";
```
Comment (jeesunikim) [inline src/types/types.ts:368]: I only see `status`, `ts`, `repository`, and `commit` from `ContractValidationStatus` from the api:
```status:
          description: Validation status
          type: string
          enum: [ verified,unverified ]
        ts:
          description: Validation timestamp
          $ref: '#/components/schemas/Timestamp'
        repository:
          type: string
          description: GitHub repository URL
          example: 'https://github.com/reflector-network/reflector-contract'
        commit:
          type: string
          description: Commit hash
          example: '391f37e39a849ddf7543a5d7f1488e055811cb68'```

File: src/types/types.ts (hunk 2 above)
Comment (quietbits) [reply src/types/types.ts:380]: `functions` is returned in the API response but not documented. I tried to combine both versions for types.
image

File: src/types/types.ts (hunk 3 above)
Comment (quietbits) [reply src/types/types.ts:368]: Same as above, it's in the API response. Thanks for checking! :raised_hands:

**Review by jeesunikim — APPROVED** [review]

**Pull request merged**